toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -193,10 +193,6 @@ nghttp2-1.46.0-2.cm2.aarch64.rpm
 curl-8.0.1-1.cm2.aarch64.rpm
 curl-devel-8.0.1-1.cm2.aarch64.rpm
 curl-libs-8.0.1-1.cm2.aarch64.rpm
-tdnf-3.2.2-4.cm2.aarch64.rpm
-tdnf-cli-libs-3.2.2-4.cm2.aarch64.rpm
-tdnf-devel-3.2.2-4.cm2.aarch64.rpm
-tdnf-plugin-repogpgcheck-3.2.2-4.cm2.aarch64.rpm
```

Comment:
I'd only update the package version but leave the lines where they are. I think we have a tool, which re-builds this file if needed, and it dictates the order of packages. Also, there's an order in which we want these packages installed and I'm not sure, if this is the source of this order. Might want to double-check with @anphel31.

Reply:
The build order needed to be updated for tdnf due to the change in dependencies for the new tdnf version that was causing a build error.